clients/common/rocsparse_parse_data.cpp

```diff
@@ -264,48 +264,6 @@ bool rocsparse_parse_data(int& argc, char** argv, const std::string& default_fil
             }
             rocsparse_reproducibility_t::instance().config().set_info_level(r_level);
         }
-        else if(!strcmp(argv[i], "--rocsparse-clients-enable-test-debug-arguments"))
```

Comment:
I might be missing where this is moved to, but do we want to delete this?

Reply:
It is moved to the main of rocsparse-test.

Reply:
Ok if I am understanding this, we delete this because either the options are enabled by default or we use environment variable to enable them and no longer pass it as commandline arugments that would be processed here?

Reply:
See comments below.